### PR TITLE
Simpler indices in force computations.

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: physics/ephemeris.hpp [hunk 1]
```diff
@@ -246,15 +246,16 @@ class Ephemeris {
...
-  // indices [b2_begin, b2_end[ in the |positions| and |accelerations| arrays).
-  // The template parameters specify what we know about the bodies, and
-  // therefore what forces apply.
+  // indices [b2_begin, b2_end[ in the |bodies2|, |positions| and
+  // |accelerations| arrays).  The template parameters specify what we know
+  // about the bodies, and therefore what forces apply.
   template<bool body1_is_oblate,
-           bool body2_is_oblate>
+           bool body2_is_oblate,
+           typename MassiveBodyConstPtr>
```
Comment (eggrobin) [inline physics/ephemeris.hpp]: Comment the templatization.

File: physics/ephemeris.hpp [hunk 2]
```diff
@@ -246,15 +246,16 @@ class Ephemeris {
 
   // Computes the acceleration due to one body, |body1| (with index |b1| in the
```
Comment (eggrobin) [inline physics/ephemeris.hpp]: both accelerations

File: physics/ephemeris.hpp (hunk 1 above)
Comment (pleroy) [inline physics/ephemeris.hpp]: Done.

File: physics/ephemeris.hpp (hunk 2 above)
Comment (pleroy) [inline physics/ephemeris.hpp]: Done.

**Pull request merged**